app.js

```diff
@@ -5,6 +5,15 @@ app.get('/', function( req, res ) {
   res.send('Hello mu-javascript-template');
 } );
 
+async function deleteAnotated(){
```

Comment:
I would rather if we fix the query in the function rather than adding another one, you have a part in the line 174 that deletes the old data that I think it needs to change the graph where it deletes, I can take a look tomorrow and try to fix it if you can't.
```
DELETE WHERE {
      GRAPH <http://mu.semte.ch/graphs/public> { 
        ${annotatedArray.map((template, index) => `<${template.uri}> ext:annotated ?${index}.`).join(' ')}
      }
    };
    ```